app-directory/i18n/app/layout.tsx

```diff
@@ -0,0 +1,39 @@
+import Nav from '@vercel/examples-ui/nav'
+import Vercel from '@vercel/examples-ui/icons/vercel'
+
+import '@vercel/examples-ui/globals.css'
+
+interface Props {
+  children: React.ReactNode
+}
+
+export default async function RootLayout({ children }: Props) {
+  return (
+    <html>
```

Comment:
Hi @goncy! I stumbled upon this PR. Should Next.js possibly provide a way to read all matched segments or alternatively the `pathname` from the root layout? I guess you'd need it here too to set the `lang` attribute, right?

Many thanks!